### DISCUSSION TITLE
Is the WAV file format supported?

### DISCUSSION THREAD

**Discussion by lev666** [navidrome/navidrome#2919] [Q&A] [answered]
Good afternoon, there was an issue with the WAV format after I added the files to the server but they are not showing up in. Navidrome (WebUI) or any applications (checked on my phone). Does navidrome support WAV?

**Comment by metalheim** [comment, marked as answer]
Yes, navidrome supports wav, but wav as a Format has limited tag (metadata) support. 
You can check the metadata of a file by running ```navidrome inspect /path/to/yourfile.wav```

Probably It's not tagged, or not tagged correctly (f.e. Id3 tags in a wav file which are non-standard) and navidrome sorted the file into an "[Unknown Album]" from a "[Unknown artist]"

**Comment by lev666** [comment]
> Yes, navidrome supports wav, but wav as a Format has limited tag (metadata) support. 
> You can check the metadata of a file by running ```navidrome inspect /path/to/yourfile.wav```
> 
> Probably It's not tagged, or not tagged correctly (f.e. Id3 tags in a wav file which are non-standard) and navidrome sorted the file into an "[Unknown Album]" from a "[Unknown artist]" 

Yes, you're absolutely right. The tracks are displayed in "Unknown Album"

But also tracks are sometimes played through "random track playback"

**Reply by metalheim** [reply]
Hi @lev666 is this solved for you?
If yes, can you please mark this discussion as "answered"?

If you still have issues, i recommend re-tagging the files with a known-good tagging software (kid3, mp3tag, yate, musicbrainzpicard etc.)
